packages/optimizers/cssnano/src/CSSNanoOptimizer.js

```diff
@@ -4,13 +4,41 @@ import SourceMap from '@parcel/source-map';
 import {Optimizer} from '@parcel/plugin';
 import postcss from 'postcss';
 import cssnano from 'cssnano';
+import type {CSSNanoOptions} from 'cssnano'; // TODO the type is based on cssnano 4
+import path from 'path';
 
 export default (new Optimizer({
+  async loadConfig({config}) {
+    const configFile = await config.getConfig(
+      ['.cssnanorc', 'cssnano.config.json', 'cssnano.config.js'],
+      {
+        packageKey: 'cssnano',
+      },
+    );
+    // TODO copied from stylus package. make this a utility function
+    if (configFile) {
+      let isJavascript = path.extname(configFile.filePath) === '.js';
+      if (isJavascript) {
+        config.shouldInvalidateOnStartup();
+      }
+
+      // Resolve relative paths from config file
+      if (configFile.contents.paths) {
+        configFile.contents.paths = configFile.contents.paths.map(p =>
+          path.resolve(path.dirname(configFile.filePath), p),
+        );
+      }
```

Comment:
The only options in the config file are `plugins ` and `preset`

Reply:
Yes, how does it affect this?

Reply:
That if statement isn't needed

Reply:
I copied this part from the Sylus package. I don't know the reasoning behind this part of the code.